tests/bgp/test_bgpmon.py

```diff
@@ -109,6 +109,17 @@ def build_syn_pkt(local_addr, peer_addr):
     return exp_packet
 
 
+def test_resolve_via_default_exist(duthost):
+    """
+    Test to verify if 'ip nht resolve-via-default' and 'ipv6 nht resolve-via-default' are present in global FRR config.
+    """
+    frr_global_config = duthost.shell("vtysh -c 'show running-config'")['stdout']
```

Comment:
@Gfrom2016 we have vrf enabled, will those 2 cases pass? How to make sure even if vrf is enabled, those 2 lines are not present under vrf section?

Reply:
@ZhaohuiS good point, from the livesite issue where we found this bug, if vrf is enabled and without exiting vrf block, 'ip nht resolve-via-default' config is missing from both global and vrf block. It didn't configurated into vrf section. So I think we can cover vrf enabled scenario.